serverless/src/graphQl/permissions/rules/userOwnsProject.js

```diff
@@ -0,0 +1,21 @@
+import { rule } from 'graphql-shield'
+
+/**
+ * Checks if the user has admin privileges
+ * @returns {boolean} true if the user is valid, false otherwise
+ */
+const userOwnsProject = rule()(async (parent, args, context) => {
+  const { databaseClient, user } = context
+  const { id: userId } = user
+  const { obfuscatedId } = args
+
+  if (userId) {
```

Comment:
I think this is actually better than how we currently do it now. In my VERY specific probably only case I have three users with the same ID for all three envs I think the way this will work now is that my SIT one would have access to the UAT and prod ones. I think that is actually better it will make testing at least easier but, wanted to call that out

Reply:
I don't understand what you are saying